### ISSUE TITLE
MediaQuery.of(context).accessibleNavigation does not work on flutter web

### ISSUE THREAD

**Issue by rami-a** [flutter/flutter#67571]
It looks like this is mapped to `window.accessibilityFeatures.accessibleNavigation` at the moment but perhaps it could use `window.semanticsEnabled`?

**Comment by rami-a** [comment]
cc @yjbanov

**Comment by nturgut** [comment]
@rami-a what is the use case for `MediaQuery.of(context).accessibleNavigation`?

The reason I'm asking is, we can't always tell for 100% if assistive technology is on or off for the web engine. That is a private information that browser keeps. In order to enable semantics therefore we add an `enable accessibility` button to the DOM. If there is a screen reader users can interact with it and enable semantics. Then we change the value of `window.semanticsEnabled`. Therefore when the page initially loads the value in `window.semanticsEnabled` is not correct. 

Is syncing it to `window.semanticsEnabled` still good for `MediaQuery.of(context).accessibleNavigation` or is the first user events/page loading is critical?

**Comment by rami-a** [comment]
@nturgut that is good to know. I think that for my specific use case in https://github.com/flutter/flutter/pull/67562, it is fine to sync them since the time picker is presented as popup so it will have the right information when it is displayed. But I could see this being not as good if on first page load the value is incorrect.

Perhaps syncing it to `window.semanticsEnabled` is fine if the documentation specifies that there are some limitations? Or perhaps when the `enabled accessibility` button is tapped, that triggers a rebuild of the widget tree?

Thoughts?

**Comment by nturgut** [comment]
@ferhatb Do you think triggering rebuild on the widget tree after accessibility is enabled would be a good/feasible?

**Comment by ferhatb** [comment]
Yes it is feasible, could be handled similar to metricsChanged.
